java/src/jmri/jmrix/loconet/pr2/PR2SystemConnectionMemo.java

```diff
@@ -111,7 +111,7 @@ public boolean provides(Class<?> type) {
             return getProgrammerManager().isAddressedModePossible();
         }
 
-        return false;
+        return super.provides(type);
```

Comment:
@bobjacobsen will this cause issues with PR2 connections?

we have a "MustInvokeSuper" annotation on the jmri.jmrix.SystemConnectionMemo.provides() method now, but this is a granchild of that class.  This class doesn't provide everything that the standard LocoNet SystemConnectionMemo provides.

Since this class provides an addrssed programmer, the DccConsistManager (which uses operations mode programming) can be installed.  After this PR,  mri.jmrix.SystemConnectionMemo handles that.

Reply:
I think that the `super.provides(type)` is a good change, but perhaps I'm not understanding the question.

Reply:
The question here was is this change going to enable anything provided by the LoconetSystemConnectionMemo that isn't supposed to be enabled on the PR2

Reply:
Ah, thanks, understand.  Yes, with this change the parent will be providing managers for lights, turnouts, sensors, etc that are not appropriate.  So the "false" should be restored. Sorry to not catch that.